doc/changes/changes_4.2.0.md

```diff
@@ -0,0 +1,17 @@
+# SQL Statement Builder 4.2.0, released 2020-10-??
+
+Code Name: LIKE predicate support
+
+## Features / Enhancements
+ 
+* #91: Added support for LIKE predicate.
+
+## Dependency updates
+
+* Added org.junit.jupiter:junit-jupiter:5.7.0
+* Updated org.mockito:mockito-core:jar:3.3.3 to version 3.5.13
```

Comment:
wrong format.
In the RR template (https://github.com/exasol/release-robot/blob/master/doc/user_guide/changes_file_template.md) the following format is suggested:
`Updated <dependency groupId>:<dependency artifactId> from to `
I wrote a converter:
[converter.zip](https://github.com/exasol/sql-statement-builder/files/5364790/converter.zip)
Just place the two scripts in the folder above your projects root, and run from the project's root `../dependencyUpdate.sh`

Reply:
The format is not really checked somewhere and it's not stable yet, so we can change it to this one because it's easier to copy-paste after the updates

Reply:
well but It is not readable. The `:jar` has no meaning to customers. Using the script I sent you can still copy & paste.

Reply:
I also was not sure about `jar`, probably is can be removed. But the rest can be changed to this one, as it's already used in the plugin